pkg/tagger/tagstore.go

```diff
@@ -75,6 +75,12 @@ func (s *tagStore) processTagInfo(info *collectors.TagInfo) error {
 
 	storedTags.Lock()
 	defer storedTags.Unlock()
+	_, found := storedTags.lowCardTags[info.Source]
+	if found && info.CacheMiss {
+		// check if the source tags is already present for this entry
+		// Only check once since we always write all cardinality tag levels.
+		return log.Warnf("processTagInfo err: try to overwrite an existing entry with and empty cache-miss entry, info.Source: %s, info.Entity: %s", info.Source, info.Entity)
```

Comment:
Might `warn` be too verbose? Given that we're protecting against order of execution, I could see this triggering often, maybe `info` or `debug` is more appropriate?

Reply:
right! I putted at `log.Tracef()` since it is now a normal code path.